about.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/nav-bar.css">
+    <link rel="stylesheet" href="./styles/about-styles.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <title>Global Training Courses - About Us</title>
+</head>
+<body>
+  <header>
+    <nav class="nav-mobile">
+      <img class="menu-button" id="menu-button" src="./assets/hamburger-menu.svg" alt="menu">
+      <div class="menu" id="menu">
+        <div>
+          <img id="close-button" src="./assets/close.png" alt="close">
+        </div>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="index.html#topics">Topics</a></li>
+          <li><a href="index.html#courses">Courses</a></li>
+          <li><a href="#">Log In</a></li>
+          <li><a href="#">Sign Up</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="nav-desktop">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#topics">Topics</a></li>
+        <li><a href="#courses">Courses</a></li>
+        <li><a href="#">Log In</a></li>
+        <li><a href="#">Sign Up</a></li>
+      </ul>
+    </nav>
+    <h1>GLOBAL TRAINING COURSES</h1>
+    <p class="about-info">Global Training Courses is a worldwide leader in IT and technology training, helping individuals and organizations build the skills they need to succeed in an ever-changing world.<br><br>
+      With an international network of offices and training facilities, Global Knowledge has the unique flexibility to deliver a broad portfolio of courses on over 100 countries online, in classrooms, and through a worldwide partner network.<br><br>
+      Established in 1995, Global Training Courses has over 1000 employees worldwide and enables the success of more than 200,000 tech professionals each year.<br><br>
+      In June 2021, Global Training Courses merged with Skillsoft to create the world’s leading corporate learning company.</p>
+    <div class="contact-info">
+      <p>Customer support, sales assistance and enterprise clients available.</p>
+      <p>1-800-268-7737</p>
+      <p>contact@skillsoft.com</p>
+    </div>
+  </header>
+  <main>
```

Comment:
After you create your logo please try to add it here as the design provided 
![image](https://user-images.githubusercontent.com/61790587/164915107-5fcedf32-208e-49c0-867e-af0b1471387b.png)

Reply:
As you can see, i changed all the content of the page, so I don't have a logo to put in the nav bar